loopy/target/c/codegen/expression.py

```diff
@@ -418,7 +418,7 @@ def map_constant(self, expr, type_context):
                 return Literal(repr(np.float32(expr))+"f")
             elif type_context == "d":
                 return Literal(repr(float(expr)))
-            elif type_context == "i":
```

Comment:
There's a spot like this in `targets.ispc` that's missed by this.

Reply:
Thanks for catching that. Fixed in  fd307f2.